ParkingApplication.api/Models/UserModel.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ParkingApplication.Models;
+
+public class UserModel
+{
+    
+    
```

Comment:
Verifica prin fisiere, ai mult prea multe linii goale intre blocurile de cod, o linie , max 2 in cazul in care vrei sa evidentezi separarea

Reply:
its done.